src/main/scala/com/datastax/sparkstress/SparkCassandraStress.scala

```diff
@@ -124,7 +127,11 @@ object SparkCassandraStress {
     }
 
     parser.parse(args, Config()) map { config =>
-      runTask(config)
+      if (config.trials > 1 && config.terminationTimeMinutes > 0) {
```

Comment:
Should we add the case where a read task is requested with terminationTimeMinutes?